### PR TITLE
Navegación de Año y Volúmenes de Publicaciones

### PR DESCRIPTION
**Pull request by beatriz878**
Navegación de Año y Volúmenes de Publicaciones

Desktop

image

Mobile 

image

### CODE REVIEW

**Comment by olmedo90** [comment]
image

image

It is not according to the figma. Check the size and transparency of the text.
👇🏼👇🏼👇🏼
image

File: src/hooks/useMediaQuery.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from 'react';
+
+type MediaQueryHook = (query: string) => boolean;
+
+const useMediaQuery: MediaQueryHook = (query) => {
+  const [matches, setMatches] = useState(false);
```
Comment (Ditmar) [inline src/hooks/useMediaQuery.tsx:6]: Really nice way to check the device good job

**Review by Ditmar — CHANGES_REQUESTED** [review]
Nice pr please review and check the error reported by Olmedo, but the code is really clear good job

File: src/hooks/useMediaQuery.tsx (hunk 1 above)
Comment (beatriz878) [reply src/hooks/useMediaQuery.tsx:6]: code details resolved in pr year and volume

Please check my pr ing

**Pull request merged**